ntbs-service/Jobs/UpdateTableCountsJob.cs

```diff
@@ -84,7 +86,7 @@ private async Task CompareRecentTableCounts()
 
             var currentValue = (int)property.GetValue(currentCount);
             var previousValue = (int)property.GetValue(previousCount);
-            if (currentValue < previousValue)
+            if (currentValue < previousValue * DecreaseMarginOfError)
```

Comment:
I think it would be worth adding logic to say that if a table count only decreases by 1 then we don't want an error - so that if one of the smaller tables does decrease then we don't get notified.

Reply:
Yeah I agree, done!